africanus/util/beams.py

```diff
@@ -158,19 +158,19 @@ def beam_grids(header):
 
     # Find the relevant axes
     for i in range(beam_axes.ndims):
-        if beam_axes.ctype[i] in ('L', 'X'):
+        if beam_axes.ctype[i] in ('L', 'X', 'px'):
```

Comment:
How about handling both lower case and upper case at once

```suggestion
        if beam_axes.ctype[i].upper() in ('L', 'X', 'PX'):
```